aten/src/ATen/core/ir.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <ATen/core/function_schema.h>
+#include <ATen/core/ivalue.h>
+#include "nomnigraph/Graph/Graph.h"
+#include "nomnigraph/Representations/ControlFlow.h"
+
+enum class IRNodeKind { Operator, Value };
+
+// Purely moveable tagged union to wrap torch::jit::{FunctionSchema,IValue}
```

Comment:
Hmmm when would you want to have a union of function schema or an IValue? One seems to be a thing that's generally available and used statically before you start the execution, the other one is a run time value.

Reply:
Also, what's the plan for this `ir.h` vs `torch/csrc/jit/ir.h`? I thought we were mostly going to use the JIT IR instead of nomnigraph?